### ISSUE TITLE
Continous change of README 

### ISSUE THREAD

**Issue by Darkfrobozz** [Darkfrobozz/inlupp4#15]
Instead of changing readme at once, it may be better to sometimes take a look and reflect what we have done. Thus, both collecting valid and relevant information in the README and keeping our eyes on the big picture.

**Comment by Grimrath** [comment]
Yes, Updating the readme should be done organically, so should testing for example. A good thing to add to the README would for example be how to create a test coverage report. How to install dependencies etc.

In an ideal world we should not consider feature 'A' implemented unless the following is also included:
- javadoc strings for methods and classes implemented in A
- README updates (if necessary) to reflect addition of A
- Test classes and methods that fully test feature A (they should also pass)

So I think we should opt (as best we can) to write tests and docs for any code we write before we commit and push.
